compiler/rustc_driver/src/lib.rs

```diff
@@ -170,17 +172,11 @@ impl<'a, 'b> RunCompiler<'a, 'b> {
         self
     }
 
-    /// Emit diagnostics to the specified location.
-    ///
-    /// Used by RLS.
-    pub fn set_emitter(&mut self, emitter: Option<Box<dyn Write + Send>>) -> &mut Self {
-        self.emitter = emitter;
-        self
-    }
-
     /// Load files from sources other than the file system.
     ///
-    /// Used by RLS.
+    /// Has no uses within this repository, but is used by bjorn3 for "hooking
+    /// rust-analyzer's VFS into rustc at some point for running rustc without
+    /// having to save". (See #102759.)
```

Comment:
It isn't used for this yet, but may be in the future.

Reply:
I have updated the comment accordingly.

Other than that, are you happy with this PR?